query/src/servers/clickhouse/clickhouse_session.rs

```diff
@@ -31,48 +26,18 @@ pub struct ClickHouseConnection;
 
 impl ClickHouseConnection {
     pub fn run_on_stream(session: SessionRef, stream: TcpStream) -> Result<()> {
-        let blocking_stream = Self::convert_stream(stream)?;
-        ClickHouseConnection::attach_session(&session, &blocking_stream)?;
-        let non_blocking_stream = TcpStream::from_std(blocking_stream)?;
         let query_executor =
             Runtime::with_worker_threads(1, Some("clickhouse-query-executor".to_string()))?;
 
         Thread::spawn(move || {
             let join_handle = query_executor.spawn(async move {
                 let interactive_worker = InteractiveWorker::create(session);
-                ClickHouseServer::run_on_stream(interactive_worker, non_blocking_stream).await
+                ClickHouseServer::run_on_stream(interactive_worker, stream).await
             });
 
             let _ = futures::executor::block_on(join_handle);
         });
 
         Ok(())
     }
-
-    fn attach_session(session: &SessionRef, blocking_stream: &std::net::TcpStream) -> Result<()> {
```

Comment:
This is related to the `kill connection statement` in MySQL. maybe we should try to test it. but i don't known to test it.

Reply:
As far as I know, we have unit test coverage this and it's using the rust MySQL driver, but I don't know why it pass. It needs to be fixed.

Reply:
This is where I got confused, I tested the kill statement specifically but everything worked fine. Perhaps I missed something?